fender.bnf

```diff
@@ -19,6 +19,7 @@ filename ::= ([a-z] | [A-Z] | [0-9] | [./_])+
 importFields ::= "*" | name ("::" importFields)? | importTargetMulti
 importTargetMulti ::= wrappedDelimitedList<"(", ")", importFields, ",", sep>
 export ::= "@export" sep ("*" | codeBody)
+plugin ::= "@plugin" sep filename 
```

Comment:
Can we have these work in the same style as imports, where you can do `::*` to just dump everything into the namespace, and otherwise pack it into a struct? You should be able to reuse some of my code from namespacing.rs.